backend/webserver/converters.py

```diff
@@ -97,13 +97,94 @@ def convert_author(self, data: dict):
     def convert_post(self, data: dict):
         raise NotImplementedError  # TODO
 
-
 class Team10Converter(Converter):
+    
     def skip_follow_check_before_sending_follow_request(self):
         return True
 
     def url_to_send_follow_request_at(self, author_url):
         return join_urls(author_url, "followers", ends_with_slash=True)
     
-    def public_posts_url(self, api_url):
-        return join_urls(api_url, "posts/public", ends_with_slash=True)
+    def send_follow_request(self, request_data):
+        converted_data = {
+            "actor": request_data["sender"]["url"]
+        }
+        return converted_data
+    
+    def expected_status_code(self, use_case_name):
+        if use_case_name == "send_follow_request":
+            return 200
+        return self.expected_status_codes.get(use_case_name, 200)
+    
+    def send_post_inbox(self, post,request: HttpRequest):
```

Comment:
double check after receiving Team 10's response to this question - https://discord.com/channels/1041933167340290129/1041933167847817239/1044438757157392445